src/types/ellipse.js

```diff
@@ -60,7 +60,12 @@ EllipseAnnotation.defaults = {
   xMax: undefined,
   yScaleID: 'y',
   yMin: undefined,
-  yMax: undefined
+  yMax: undefined,
+  // hooks
+  afterDraw: undefined,
```

Comment:
yes, but in this case, do we have really a use case where you can apply the draw logic for all annotations types?
Personally I don't see any use case for that. Instead, for events, there are because the actions to apply when a  event is coming  could be the same (i.e. invoking a link).